src/sentry/api/endpoints/project_rules.py

```diff
@@ -830,4 +833,28 @@ def post(self, request: Request, project) -> Response:
             wizard_v3=wizard_v3,
         )
 
+        # TODO add msteams and discord
+        SUPPORTED_ACTIONS_MAP = {
+            "sentry.integrations.slack.notify_action.SlackNotifyServiceAction": "sentry.integrations.slack.actions.notification.SlackNotifyServiceAction"
+        }
```

Comment:
I'd suggest that rather than a list of supported actions, we should just have `send_confirmation_notification` be a function on the base class that does nothing, and can then be overridden when you want to implement it. Probably on `EventAction`?